build.cmd

```diff
@@ -152,6 +152,14 @@ set Platform=
 if NOT "%__DotNetCliPath%" == "" goto SetupManagedBuild
 
 set "__DotNetCliPath=%__RootBinDir%\tools\cli"
+if "%__CleanBuild%"=="1" (
```

Comment:
Are you sure, these don't change often and takes quite some time to download? You can always do a `git clean -fxd .` if you really wanted super clean.